### ISSUE TITLE
Storage Location in settings not working as expected.

### ISSUE THREAD

**Issue by satscape** [obiscr/ChatGPT#273]
In the settings for this plugin, there are several 'storage locations'. They have defaulted to "/home/scott/intellij-chatgpt" on my PC (Linux). I didn't want it cluttering up my home folder, so I've changed all the storage locations to "/home/scott/.intellij-chatgpt" (note the full stop) to make it use a hidden folder instead. It is using that hidden folder, but it it still creating the default non-hidden folder and dropping the machineid file in there. It would be great if it didn't do that and use the folder I specified instead.
Thanks.

**Comment by obiscr** [comment]
Yes, it's not currently in a hidden folder, however, that part would be a bit tricky to modify. There would be a lot of correlations. This change will probably be put in a future version.
